main/src/modules/tools/opcPopulater/include/opcPopulater.h

```diff
@@ -19,6 +19,11 @@ class opcPopulater : public RFModule {
     double Y_ag;
     double Z_ag;
     double noise;
+    vector<double> spd1;
+    vector<double> spd2;
```

Comment:
`spd1` and `spd2` were probably meant for testing, but are not used anymore as far as I can see. Can you please remove all references to them?